src/Compilers/CSharp/Test/Symbol/Symbols/SymbolErrorTests.cs

```diff
@@ -15327,19 +15327,25 @@ public void CS1003ERR_InvalidFixedArrayMultipleDimensions()
             CreateCompilation(text, options: TestOptions.UnsafeReleaseDll).VerifyDiagnostics(
                 // (4,33): error CS1002: ; expected
                 //     public unsafe fixed int B[2][2];   // CS1003,CS1001,CS1519
-                Diagnostic(ErrorCode.ERR_SemicolonExpected, "["),
+                Diagnostic(ErrorCode.ERR_SemicolonExpected, "[").WithLocation(4, 33),
+                // (4,36): error CS1519: Invalid token ';' in class, record, struct, or interface member declaration
```

Comment:
>// (4,36): error CS1519: Invalid token ';' in class, record, struct, or interface member declaration [](http://example.com/codeflow?start=16&length=100)

It looks like this error is mentioned below. Are we reporting duplicate errors now?